compiler/rustc_codegen_ssa/src/mir/analyze.rs

```diff
@@ -15,13 +15,18 @@ use crate::traits::*;
 
 pub fn non_ssa_locals<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>>(
     fx: &FunctionCx<'a, 'tcx, Bx>,
+    traversal_order: &[mir::BasicBlock],
```

Comment:
Which traversal order is this? Or do we want to leave it unspecified?

Reply:
The traversal order requirements are spelled out in a comment in this function:

>  If there exists a local definition that dominates all uses of that local, the definition should be visited first. Traverse blocks in an order that is a topological sort of dominance partial order.

So we can't traverse in just any order and indeed we don't. What do you think I should call this argument instead?